components/datetime/src/fields/symbols.rs

```diff
@@ -446,6 +448,10 @@ field_type! (
         ///
         /// This works for “week of year” based calendars in which the year transition occurs on a week boundary; may differ from calendar year [`Year::Calendar`] near a year transition. This numeric year designation is used in conjunction with [`Week::WeekOfYear`], but can be used in non-Gregorian based calendar systems where week date processing is desired. The field length is interpreted in the same way as for [`Year::Calendar`].
         'Y' => WeekOf = 1,
+        /// TODO: Docs
+        'U' => Cyclic = 2,
+        /// TODO: Docs
+        'r' => RelatedIso = 3,
```

Comment:
Nit: File an issue for the TODO since it got merged without fixing. Our linting won't catch this.